.circleci/config.yml

```diff
@@ -1,4 +1,11 @@
-steps:
-  - name: microsoft/dotnet:2.1-sdk
-    entrypoint: dotnet
-    args: ['build', 'src/GlobalX.Hypermedia.AspNetCore.sln']
\ No newline at end of file
+version: 2
+jobs:
+  build:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:2.1
+    steps:
+      - checkout
+      - run: find .
+      - run: dotnet restore src/GlobalX.Hypermedia.AspNetCore.sln
```

Comment:
From .net core 2 and above you don't need to explicitly run a restore.
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack